classes/PrestaShopAutoload.php

```diff
@@ -260,7 +260,7 @@ public function dumpFile($filename, $content)
             return false;
         }
         // Ignore for filesystems that do not support umask
-        @chmod($tmpFile, 0666);
+        @chmod($tmpFile, file_exists($filename) ? fileperms($filename) : 0666 & ~umask());
```

Comment:
For someone not used to bitwise operators (like me 😛 ) the meaning of `& ~` is not obvious.
Do you think we could name a variable to make it more obvious ?

Reply:
This function is a copy of the Symfony autoloader, I just update the behavior to keep the filepermissions and get umask configuration if someone change it. (Like Symfony does)

Reply:
For the variable name, it is overkill :/